### DISCUSSION TITLE
Irradiation Parameters

### DISCUSSION THREAD

**Discussion by TomRadfahrer** [tbnobody/OpenDTU#335] [General]
In the settings of the inverter you can define 4 parameters for irradiation. I am wondering how I can use them.

**Assumption:** The parameters define the GPS coordinates of the solar panel and azimuth and inclination. The inverter uses these parameters to calculate the relative position of the sun. If the panels are facing directly to the actual position of the sun the irradiation is 100%. In the best case you achieve 100% irradiation once a day. In the morning and in the afternoon irradiation should be less than 100%. In the night irradiation should be 0%.

Ok, this is just an educated guess, not more. I found in the documentation of the Hoymiles web interface that a solar panel is configured with GPS coordinates (by locating them in a map). Additionally azimuth and inclination can be configured. This gives 4 parameters.

I tried a simple set of parameters: GPS North, GPS East, azimuth, inclination and configured "48", "11", "180" and "30". My idea was that this is the location of Munich, the solar panel is facing south and has an inclination of 30°. The result was not good: The irradiation was about 120% and changed rapidly between 105% and 130%. There are two problems: Why does irradiation exceeds 100% and why does it change rapidly? In my world the sun is moving slowly.

I tried "48.0", "11.0", "180.0" and "30.0" also, but the settings accept integer values only. 

Any idea?

**Comment by tbnobody** [comment]
Its not as high sophisticated as you maybe think... you can enter the max power of your solar panels in this text boxes (or lets say the max production power which is connected to each of the inputs). Using this values, the Irradiation values in percent on the LiveData view are calculated. It's just the current produced power divided by the total possible power entered in the settings.

**Reply by TomRadfahrer** [reply]
Sometimes live is more easy than I assume :-)
Thank you very much for your explanation

**Reply by stefan123t** [reply]
@TomRadfahrer,
I was thinking about azimuth and inclination and GPS latitude / longitude parameters as well some time ago.
We have lat / lon in the settings (at least in AhoyDTU) and there is a database PVGIS from the EU which has global radiation data for each place in Europe available. So given azimuth and inclination for your position you can simulate / calculate the expected PV yield for different angles over the year.
Maybe we could use the same data, they have two granularities online (i.e. per hour of day or per day) ?
https://joint-research-centre.ec.europa.eu/pvgis-photovoltaic-geographical-information-system/pvgis-tools_en

**Reply by TomRadfahrer** [reply]
Very interesting web site! I think this is very helpful for planning.
